tuxemon/npc.py

```diff
@@ -653,7 +653,7 @@ def network_notify_location_change(self) -> None:
     ####################################################
     #                   Monsters                       #
     ####################################################
-    def add_monster(self, monster: Monster) -> None:
+    def add_monster(self, monster: Monster, slot: int) -> None:
```

Comment:
I would have gone with the following, then you don't need to change the existing calls.

`
def add_monster(self, monster: Monster, slot: int = None) -> None:
`

Reply:
I guess this is ok, at least fix the typehint.

Reply:
It sound good, I'll update